terraform/modules/kubernetes/kubernetes.tf

```diff
@@ -558,6 +657,96 @@ resource "kubernetes_deployment" "aggregate" {
     }
   }
 }
+
+resource "kubernetes_manifest" "aggregate_queue_depth_metric" {
```

Comment:
There's no GCP equivalent to this. Does the GCP metrics adapter automagically create the manifest?

Reply:
This reminds me of network configuration, which mostlu Just Works in GKE, but EKS requires you to jump through a ton of hoops to construct subnets and security groups and so on. Anyway, I agree with your approach here.